services/api/kalvi/api/views/auth.py

```diff
@@ -1,15 +1,87 @@
 from rest_framework.views import APIView
 from rest_framework.response import Response
+from rest_framework import status
+from db.serializers import SignUpEndPointSerializer, SignInEndPointSerializer, UserProfileSerialiser, UserChangePasswordSerialiser, SendPasswordResetEmailSerializer, UserPasswordResetSerializer
+from django.contrib.auth import authenticate
+from db.renderer import UserRenderer
+from rest_framework_simplejwt.tokens import RefreshToken
+from rest_framework.permissions import IsAuthenticated
+from django.utils.encoding import smart_str, force_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from db.models import User
 
+# Generate stateless token for User
+def get_tokens_for_user(user):
+  refresh = RefreshToken.for_user(user)
+  return {
+      'refresh': str(refresh),
```

Comment:
Pls check if the str casting is really needed.

Reply:
I checked and its needed

Reply:
I did check and it was throwing error without typecasting.